pkgs/development/python-modules/pyahocorasick/default.nix

```diff
@@ -4,6 +4,7 @@
 , fetchpatch
 , pytestCheckHook
 , pythonOlder
+, buildAsUnicode ? true
```

Comment:
```suggestion
```
That is mostly forbidden in pythonPackages due to pip not handling multiple variants of the same package.